promise-it-wont-hurt-solutions-master/src/03-reject-a-promise.js

```diff
@@ -0,0 +1,9 @@
+
+var promise = new Promise(function (fulfill, reject) {
+    setTimeout( () => {
+      reject('REJECTED!');
+
+    }, 3000);
+  });
+
+  promise.then(console.log);
```

Comment:
The first parameter is the fulfillment handler, and the second parameter is the rejection handler, which is not called here, so the error won't be caught properly.